playbooks/robusta_playbooks/job_actions.py

```diff
@@ -236,6 +236,23 @@ def job_info_enricher(event: JobEvent):
     event.add_enrichment([table_block])
 
 
+@action
+def delete_job(event: JobEvent):
```

Comment:
maybe add to the light actions list, this could be useful for frontend

Reply:
good idea

Reply:
done